tests/filestore-alert-log/README.md

```diff
@@ -0,0 +1,7 @@
+# Description
+
+Test alert event with filestore
+
+# PCAP
+
+The pcap comes from test filestore-filecontainer-http
```

Comment:
We can avoid copying the pcap file here by using the `pcap` feature in `test.yaml`. e.g. https://github.com/OISF/suricata-verify/pull/1128/files#diff-43a344381d7ddf9d8a334ece41d352d6a338438cc1bd465b54d24c64858c099dR1

Reply:
Nice, doing it

Is there a ticket for CI to catch new pcap being a duplicate ?

Reply:
I don't believe so.. I could make one.